spec/classes.md

```diff
@@ -2900,26 +2909,76 @@ The following example:
 ```csharp
 public class ReadOnlyPoint
 {
-	public int X { get; }
-	public int Y { get; }
-	public ReadOnlyPoint(int x, int y) { X = x; Y = y; }
+    public int X { get; }
+    public int Y { get; }
+    public ReadOnlyPoint(int x, int y) { X = x; Y = y; }
 }
 ```
 is equivalent to the following declaration:
 ```csharp
 public class ReadOnlyPoint
 {
-	private readonly int __x;
-	private readonly int __y;
-	public int X { get { return __x; } }
-	public int Y { get { return __y; } }
+    private readonly int __x;
+    private readonly int __y;
+    public int X { get { return __x; } }
+    public int Y { get { return __y; } }
     public ReadOnlyPoint(int x, int y) { __x = x; __y = y; }
 }
 ```
 
 Notice that the assignments to the readonly field are legal, because they occur within the constructor.
 
 
+The following example:
+```csharp
+// No 'field' symbol in scope.
+public class Point
+{
+    public int X { get; set; }
+    public int Y { get; set; }
+}
+```
+is equivalent to the following declaration:
+```csharp
+// No 'field' symbol in scope.
+public class Point
+{
+    public int X { get { return field; } set { field = value; } }
+    public int Y { get { return field; } set { field = value; } }
+}
+```
+which is equivalent to:
+```csharp
+// No 'field' symbol in scope.
+public class Point
+{
+    private int __x;
+    private int __y;
+    public int X { get { return __x; } set { __x = value; } }
+    public int Y { get { return __y; } set { __y = value; } }
+}
+```
+
+The following example:
+```csharp
+// No 'field' symbol in scope.
+public class LazyInit
+{
+    public string Value => field ??= ComputeValue();
```

Comment:
Happy to see we don't need `auto` modifier :)

Is there any special meaning for `@field`?

Reply:
yes.  it would no longer be a keyword (contextual or otherwise) so would not count.  this falls out from our definitions around keywords/identifiers and whatnot.

Reply:
> Happy to see we don't need auto modifier :)

LDM still to decide on this.  it will come back to if we think it is critical to be able to tell if somethign is an autoprop without having to do examination of the bodies.  My perspective is that it is not critical and it's acceptable to have no top-level sigil indicating this information.